### DISCUSSION TITLE
Has anyone successfully gotten the AWS SSO integration working?

### DISCUSSION THREAD

**Discussion by StevenMonty** [goauthentik/authentik#4709] [Q&A]
Title says most of it, but following the docs step-by-step has led me with hours of debugging with no real progress. The main error message I've gotten my my cloudtrail logs are about the audience not matching up but I've found the entire process very frustrating to debug. Any insight would be greatly appreciated! Thanks in advance :)

**Comment by timvw** [comment]
Yes, for me it's working now.

Struggled for a long time, followed the steps in https://github.com/goauthentik/authentik/issues/4554, and was finally able to put all required config in terraform...
